include/boost/graph/bc_clustering.hpp

```diff
@@ -147,14 +147,14 @@ betweenness_centrality_clustering(MutableGraph& g, Done done,
 
 /**
  * \overload
- */ 
+ */
 template<typename MutableGraph, typename Done>
 void
 betweenness_centrality_clustering(MutableGraph& g, Done done)
 {
   typedef typename Done::centrality_type centrality_type;
   std::vector<centrality_type> edge_centrality(num_edges(g));
-  betweenness_centrality_clustering(g, done, 
+  betweenness_centrality_clustering(g, done,
```

Comment:
Another 2 files with just whitespace changes: with over 200 changed files and a mix of substantive and formatting changes this is getting rather hard going.   Please can we split this into smaller orthogonal changes?

Stopping at this point...